jest.config.js

```diff
@@ -2,7 +2,6 @@ module.exports = {
     rootDir: __dirname,
     verbose: false,
     silent: false,
-    browser: false,
```

Comment:
You're using this in a browser now?

Reply:
nah.... this config value is just deprecated